public/static/docs/command-reference/config.md

```diff
@@ -22,9 +22,9 @@ takes a config option `name` (a section and a key, separated by a dot) and its
 This command reads and updates the DVC configuration files. By default (if none
 of `--local`, `--global`, or `--system` is provided) a project's config
 (`.dvc/config`) file is read or modified. This file is by default meant to be
-under Git control and should not contain sensitive and/or user-specific
-information (passwords, SSH keys, etc). Use `--local`, `--global`, or `--system`
-options instead to override project's settings, for sensitive, or user-specific
+tracked by Git and should not contain sensitive and/or user-specific information
```

Comment:
Not sure about "tracked by Git" but at least has ~20k results on Google, vs ~4k of "under Git control"

Reply:
I guess it's fine? "track"-like words are common in https://git-scm.com/docs/git-add (whereas no instances of "control" are found).

Reply:
both fine with, I would not try to be very formal about this ... every formality like this should have a strong reason/ground ... in this case it's quite easy to understand both I would say

Reply:
Its not about formality in this case. "Under X control" sounds strange, as discussed in #719